lib/autoupdate/agent/updater.go

```diff
@@ -762,7 +764,9 @@ func (u *Updater) UnlinkPackage(ctx context.Context) error {
 	if err := u.Installer.UnlinkSystem(ctx); err != nil {
 		return trace.Errorf("failed to unlink system package installation: %w", err)
 	}
-	if err := u.Process.Sync(ctx); err != nil {
+	if err := u.Process.Sync(ctx); errors.Is(err, ErrNotSupported) {
+		u.Log.WarnContext(ctx, "SystemD is not installed, skip sync")
```

Comment:
```suggestion
		u.Log.WarnContext(ctx, "Systemd is not installed. Skipping sync.")
```
Just for consistency with the other logs. (SystemD is probably more correct.)